Yamb/RegisterViewController.swift

```diff
@@ -0,0 +1,100 @@
+//
+//  RegisterViewController.swift
+//  Yamb
+//
+//  Created by Ana Peshevska on 1.7.22.
+//
+
+import UIKit
+
+class RegisterViewController: UIViewController {
+    
+    lazy var titleLabel: UILabel = {
+        var label = UILabel()
+        label.textAlignment = .center
+        label.text = "Register"
+        label.font = .systemFont(ofSize: 24, weight: .semibold)
+        label.snp.makeConstraints { make in
+            make.height.equalTo(50)
+            make.width.equalTo(100)
+        }
+        return label
+    }()
+    
+    lazy var nameField: CustomLoginTextField = {
+        var nameField = CustomLoginTextField()
+        nameField.placeholder = "Name"
+        return nameField
+    }()
+    
+    lazy var surnameField: CustomLoginTextField = {
+        var surnameField = CustomLoginTextField()
+        surnameField.placeholder = "Surname"
+        return surnameField
+    }()
+    
+    lazy var emailField: CustomLoginTextField = {
+        var emailField = CustomLoginTextField()
+        emailField.placeholder = "Email Address"
+        return emailField
+    }()
+    
+    lazy var passwordField: CustomLoginTextField = {
+        var passField = CustomLoginTextField()
+        passField.placeholder = "Password"
+        passField.isSecureTextEntry = true
+        return passField
+    }()
+    
+    lazy var confirmPasswordField: CustomLoginTextField = {
+        var passField = CustomLoginTextField()
+        passField.placeholder = "Confirm password"
+        passField.isSecureTextEntry = true
+        return passField
+    }()
+    
+    lazy var registerButton: CustomLoginButton = {
+        var button = CustomLoginButton()
+        button.setTitle("Continue", for: .normal)
+        button.addTarget(self, action: #selector(onRegister), for: .touchUpInside)
+        return button
+    }()
+    
+    lazy var allFieldsRequired: CustomErrorLabel = {
+        var label = CustomErrorLabel()
+        label.text = "All fields are required"
```

Comment:
try adding the texts and the placeholders to the initializer of the custom classes as optional params, it would cut down on a lot of code. you would have one liners something like `lazy var allFieldsRequired = CustomErrorLabel(text: All fields are required)`